src/api/nft.ts

```diff
@@ -105,23 +105,30 @@ export class Nft extends BaseNft {
   }
 
   /** @internal */
-  static fromResponse(ownedNft: RawNft, contractAddress: string): Nft {
+  static fromResponse(nft: RawNft, contractAddress: string): Nft {
     return new Nft(
       contractAddress,
       // We have to normalize the token id here since the backend sometimes
       // returns the token ID as a hex string and sometimes as an integer string.
-      BigNumber.from(ownedNft.id.tokenId).toString(),
-      ownedNft.id.tokenMetadata?.tokenType ?? NftTokenType.UNKNOWN,
-      ownedNft.title,
-      ownedNft.description,
-      ownedNft.timeLastUpdated,
-      ownedNft.tokenUri,
-      ownedNft.media,
-      ownedNft.metadata,
-      ownedNft.error
+      BigNumber.from(nft.id.tokenId).toString(),
+      nft.id.tokenMetadata?.tokenType ?? NftTokenType.UNKNOWN,
+      nft.title,
+      nft.description,
+      nft.timeLastUpdated,
+      nft.tokenUri,
+      nft.media,
+      nft.metadata,
+      nft.error
     );
   }
 
+  /** @internal */
+  private static parseDescription(description: string | Array<string>): string {
+    return typeof description === 'string'
+      ? description
+      : description.join(' ');
```

Comment:
random nit, stripping the description of whitespaces? not sure if that's necessary

Reply:
I'm not stripping the whitespace here, just adding a space in between each array element since the example NFT doesn't add any spacing.